libs/domain/search/pagination/src/pagination.schema.ts

```diff
@@ -0,0 +1,22 @@
+import { ContentComponentSchema } from '@spryker-oryx/experience';
+import { FormFieldType } from '@spryker-oryx/form';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { PaginationComponent } from './pagination.component';
+
+export const paginationComponentSchema: ContentComponentSchema<PaginationComponent> =
+  {
+    name: 'Pagination',
+    group: 'Search',
+    icon: IconTypes.ArrowsOutward,
+    options: {
+      max: {
+        type: FormFieldType.Number,
+        label: 'Max number of visible pages',
+        width: 100,
+      },
+      enableControls: {
+        type: FormFieldType.Boolean,
+        label: 'Enable controls',
```

Comment:
We've added resolving a standard name from the type name (splitting camel case and add a space). We should avoid label for most of the field definitions.

Reply:
please fix everywhere